dashboard/lib/clients/lti_advantage_client.rb

```diff
@@ -1,31 +1,32 @@
 class Clients::LtiAdvantageClient
   include LtiAccessToken
 
+  PAGE_LIMIT = 100
+
   def initialize(client_id, issuer)
     raise ArgumentError unless client_id && issuer
     @client_id = client_id
     @issuer = issuer
+    @page_limit = PAGE_LIMIT
```

Comment:
Since we weren't overriding this via the method parameter, I thought it might be safer to declare it as a const here. It's also cleaner than passing it down to the `build_uri` method